app/lib/filters/status.rb

```diff
@@ -3,13 +3,20 @@
 class Filters::Status < Filters::Base
   def apply
     return scope if statuses.empty?
-
     scope.where(status: statuses)
   end
 
   private
 
   def statuses
-    Array(params[:statuses]).reject(&:blank?)
+    Array(params[:statuses])
+      .reject(&:blank?)
+      .flat_map do |status|
+        if %w[initial_assessment assessment_in_progress].include?(status)
```

Comment:
```suggestion
        if status == "assessment_in_progress"
```

I don't think this will ever come through as `initial_assessment` anymore as we've removed it from the list of statuses: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1332/files#diff-e5bd83cce46d654bc07bb75cb1ba83151390557e0dd03c02798ab6b3368b8960R41